src/node_bindings/era_test_node.rs

```diff
@@ -367,48 +367,41 @@ impl EraTestNode {
                 break;
             }
 
-            if line.contains("Private Key: ") {
-                // Questionable but OK.
-                let key_str = line
-                    .split("0x")
-                    .last()
-                    .ok_or(EraTestNodeError::ParsePrivateKeyError)?
-                    .trim();
-                let key_hex = hex::decode(key_str).map_err(EraTestNodeError::FromHexError)?;
-                let key = K256SecretKey::from_bytes((&key_hex[..]).into())
-                    .map_err(|_| EraTestNodeError::DeserializePrivateKeyError)?;
-                addresses.push(Address::from_public_key(
-                    SigningKey::from(&key).verifying_key(),
-                ));
-                private_keys.push(key);
-            }
-
-            // if line.starts_with("Private Keys") {
-            //     is_private_key = true;
-            // }
-
-            // if is_private_key && line.starts_with('(') {
-            //     let key_str = line
-            //         .split("0x")
-            //         .last()
-            //         .ok_or(EraTestNodeError::ParsePrivateKeyError)?
-            //         .trim();
-            //     let key_hex = hex::decode(key_str).map_err(EraTestNodeError::FromHexError)?;
-            //     let key = K256SecretKey::from_bytes((&key_hex[..]).into())
-            //         .map_err(|_| EraTestNodeError::DeserializePrivateKeyError)?;
-            //     addresses.push(Address::from_public_key(
-            //         SigningKey::from(&key).verifying_key(),
-            //     ));
-            //     private_keys.push(key);
-            // }
-
-            if let Some(start_chain_id) = line.find("L2ChainId") {
-                // <Starting network with chain id: L2ChainId(260)>
-                // Account for parenthesis.
-                let start = start_chain_id + "L2ChainId".len() + 1;
-                let end = line.len() - 1;
-                let rest = &line[start..end];
-                if let Ok(chain) = rest.split_whitespace().next().unwrap_or("").parse::<u64>() {
+            // Questionable but OK.
+            // Start the internal loop to go over the private keys
+            if line.contains("Private Keys") {
```

Comment:
The old PK logs looked like this:
```
...
08:53:53  INFO Private Key: 0x3d3cbc973389cb26f657686445bcc75662b415b656078503592ac8c1abb8810e
...
08:53:53  INFO Private Key: 0x509ca2e9e6acf0ba086477910950125e698d4ea70fa6f63e000c5a22bda9361c
...
```
the new logs look the following way:
```
...
08:54:29  INFO Private Keys
08:54:29  INFO ========================
08:54:29  INFO (0) 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
08:54:29  INFO (1) 0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
08:54:29  INFO (2) 0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a
08:54:29  INFO (3) 0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6
08:54:29  INFO (4) 0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a
08:54:29  INFO (5) 0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba
08:54:29  INFO (6) 0x92db14e403b83dfe3df233f83dfa3a0d7096f21ca9b0d6d6b8d88b2b4ec1564e
08:54:29  INFO (7) 0x4bbbf85ce3377467afe5d46f804f221813b2bb87f24d81f60f1fcdbf7cbf4356
08:54:29  INFO (8) 0xdbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97
08:54:29  INFO (9) 0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6


08:54:29  INFO Wallet
...
```

so now it's a separate section with private keys.